tests/test_sklearn_voting_regressor_converter.py

```diff
@@ -0,0 +1,80 @@
+"""Tests GLMRegressor converter."""
```

Comment:
Typo here, VotingRegressor not GLMRegressor.

Reply:
Done